### PR TITLE
fix: support minimal top bar for onboarding screens

### PR DESCRIPTION
**Pull request by raquelmsmith**
## Problem

The Products and Onboarding scenes used the std navigation, but this was distracting for people because most of the buttons don't work on these scenes.

## Changes

Before:

image

After:

image

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

I didn't see a good way to do a story for just the minimal nav since it's computed based on the scene that is showing. So i instead added a story for the Products scene, which does use this minimal nav bar, which would change if the top bar were inadvertently changed.

I also had to update useFeatureFlags so it could specify the flag value in the case of multivariate flags, so that works now too!

### CODE REVIEW

**Comment by raquelmsmith** [comment]
@mariusandra you are the lucky recipient of my random review request :)

File: frontend/__snapshots__/scenes-app-products--products-scene.png
Comment (mariusandra) [inline frontend/__snapshots__/scenes-app-products--products-scene.png:1]: Somehow, this snapshot leaves me feeling empty.

**Review by mariusandra — APPROVED** [review]
Looks good to me, except that snapshot... however it might be better to tag @Twixes , @thmsobrmlr or anyone else from the ~~Frontend~~ NoteForce team for UI 👀

**Review by thmsobrmlr — APPROVED** [review]
This looks good to me as well.. except for the snapshot :) Don't know what is going on with that.

Also I've left a comment inline for what I think would be more standard kea usage. 

Regarding PostHog 3000: We're planning on releasing parts of the UI update soon and this will need some changes/updates as well. We (that is @PostHog/noteforce-3000) will take care of that and loop you in accordingly.

File: frontend/src/lib/logic/featureFlagLogic.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ function notifyFlagIfNeeded(flag: string, flagState: string | boolean | undefine
 
 function getPersistedFeatureFlags(appContext: AppContext | undefined = getAppContext()): FeatureFlagsSet {
     const persistedFeatureFlags = appContext?.persisted_feature_flags || []
-    return Object.fromEntries(persistedFeatureFlags.map((f) => [f, true]))
+    return Object.fromEntries(persistedFeatureFlags.map((f) => (Array.isArray(f) ? f : [f, true])))
```
Comment (neilkakkar) [inline frontend/src/lib/logic/featureFlagLogic.ts:23]: you can now default the `persistedFeatureFlags` to be an object (Record<string, string | boolean>) and get rid of the whole `Object.fromEntries()` schenanigans here and simplify things everywhere else too.

Just add `true` as value to previous default persisted flags

**Comment by raquelmsmith** [comment]
This is so frustrating, why is the snapshot blank??

**Comment by raquelmsmith** [comment]
i can't figure out the issue with the stories rn but need to get this out so will make the story changes in another pr.

**Pull request merged**